worlds/pokemon_rb/rom.py

```diff
@@ -545,7 +545,7 @@ def generate_output(self, output_directory: str):
         mons.sort(key=lambda mon: 0 if mon == self.world.get_location("Pallet Town - Starter 2", self.player).item.name
                   else 1 if mon == self.world.get_location("Pallet Town - Starter 1", self.player).item.name else
                   2 if mon == self.world.get_location("Pallet Town - Starter 3", self.player).item.name else 3)
-    write_bytes(data, encode_text(self.world.seed_name, 20, True), rom_addresses['Title_Seed'])
+    write_bytes(data, encode_text(self.world.seed_name[-20:], 20, True), rom_addresses['Title_Seed'])
```

Comment:
it's 20 here, but 21 in the lua?

Reply:
I meant to remove this part. Now I don't know why it didn't fail

Reply:
Oh, this is for the seed number displayed on the title screen in game. There isn't room for a 21st character